canonical/ubuntu.com
# switch steps 1 and 2 of server download

## Done

- Swapped steps 1 and 2 of /download/server according to copy doc: https://docs.google.com/document/d/1Qob4Ni_y1ffIdSWqaw2xH0dxiHZQmkRGP5nuoBhIfyM/edit#heading=h.w944gtee0h1h

## QA

- Visit https://ubuntu-com-11930.demos.haus/download/server
- See that step 1 is now "Manual server installation" and step 2 is now "Instant Ubuntu VMs", and make sure you can navigate back and forth between the steps

## Issue / Card

Fixes https://github.com/canonical/web-squad/issues/5235
